config/application.rb

```diff
@@ -29,7 +29,5 @@ class Application < Rails::Application
     config.time_zone = 'London'
 
     config.active_record.raise_in_transactional_callbacks = true
-    config.unsubscribe_url =
-      '<https://www.prisonvisits.service.gov.uk/unsubscribe>'
```

Comment:
We should also set this address in the `List-Unsubscribe` header when we send emails. That will probably have to be in a `before_action` in order for the URL method to work.